content/challenges/medium/8 - Is Isogram/func.js

```diff
@@ -12,5 +12,27 @@
  * @returns {boolean}
  */
 module.exports = (str) => {
-    // your code here
-}
+    // str = str.toLowerCase();
+    // for (let i = 0; i < str.length; i++) {
+    //     for (let j = i + 1; j < str.length; j++) {
+    //         if (str[i] === str[j]) {
+    //             return false;
+    //         }
+    //     }
+    // }
+    // return true;
+
+    // As i was checking for styling errors i realized this was a mutating function. so i changed it below
+
+    const newStr = str.toLowerCase();
+
+    for (let i = 0; i < newStr.length; i++) {
+        for (let j = i + 1; j < newStr.length; j++) {
+            if (newStr[i] === newStr[j]) {
+                return false;
+            }
+        }
+    }
+
+    return true;
```

Comment:
This works. Take a look at `Set()` to see if there is a way to make this even shorter/easier to read.